torch/quantization/quantize.py

```diff
@@ -260,6 +280,9 @@ def convert(module, mapping=DEFAULT_MODULE_MAPPING):
 
     return module_swapped
 
+def convert_dynamic(module, mapping=DEFAULT_DYNAMIC_MODULE_MAPPING):
```

Comment:
I don't think we need to expose it. Since majority of folks would be calling just `quantize_dynamic` (which sets observers), there's less need in this toplevel API. (note that in PTQ or QAT there's a higher need in separate apis as user needs to insert training/eval loop in the middle)

Reply:
Similarly, we do not need to expose prepare_dynamic as well. Quantize_dynamic is sufficient.

Reply:
Right: this is mainly for the unit test purposes, e.g., https://github.com/pytorch/pytorch/pull/23128/files/66528008d7b9bca74fa6817b77d9863d1ce2bb1d#diff-aceee327fb8c71ff9723e11597f7cbc7R298-R303. In the unit test, we need the `prepare_dynamic` and `convert_dynamic` for verifying the function correctness. Maybe we should just re-use `prepare_dynamic` and `convert`, and add an argument for `use_dynamic` to specially handle the dynamic quant cases?

Reply:
we can put these functions in common_quantization.py

Reply:
Put these functions in `common_quantization.py`, only for the purpose of the unit test.